recipe/meta.yaml

```
@@ -20,16 +20,16 @@ requirements:
     - python >=3.7
     - pip
     - pyct >=0.4.4
-    - param >=1.7.0,<2.0
+    - param >=1.12.0,<3.0
     - setuptools >=30.3.0
   run:
     - python >=3.7
-    - param >=1.9.3,<2.0
+    - param >=1.12.0,<3.0
     - panel >=0.13.1
     - numpy >=1.0
     - pandas >=0.20.0
     - matplotlib-base >=3.0
-    - bokeh >=1.1.0,<3.0
+    - bokeh >=2.4.0
```

It's a bit of a mess how Bokeh is pinned in `holoviews/setup.py` :/ We have in the extras:

- `tests_core`: no pin
- `recommended`: >= 2.4.3
- `doc`: >2.2

Probably the one that makes the most sense in the conda context is in `recommended`. Though I don't know ahy it's `2.4.3` specifically, there aren't much details in https://github.com/holoviz/holoviews/commit/cc29c1f9ae18860cb26f7b4c22e9601e2ae1134b.

Also, there was an upper pin before in the recipe: `bokeh >=1.1.0,<3.0`. I'd imagine that given past experience and how tightly HoloViews depends on Bokeh, it's likely that a future Bokeh 4.0 version would break HoloViews in some ways. Now Bokeh 4.0 is certainly not going to happen any time soon, so no upper pinning now wouldn't be a problem for a while; we know how conda works and if we add an upper pin in the future, it'll try to find an earlier version that has no upper pin, at which data you have to pach `repodata.json`.

So to be on the safe side I'd suggest:
```suggestion
    - bokeh >=2.4.3,<4.0
```

---

I'm not sure if it is HoloViews job to upper pin to Bokeh anymore. I think it is Panel's job. 

The breakage can also happen in a minor release or even a patch release if we are unlucky. 

I chose the lower pin because that is what Panel 0.13.1 had set. But will bump it up to 2.4.3.

---

```suggestion
    - bokeh >=2.4.3
```

---

```suggestion
    - bokeh >=2.4.3
```

---

Agree with @Hoxbro